source/administration/minio-console.rst

```diff
@@ -28,6 +28,28 @@ You can use the MinIO Console for administration tasks like Identity and Access
 The MinIO Console is embedded as part of the MinIO Server. 
 You can also deploy a standalone MinIO Console using the instructions in the :minio-git:`github repository <console>`.
 
+Supported Browsers
+~~~~~~~~~~~~~~~~~~
+
+MinIO Console runs on a variety of current, stable release browsers.
+
+MinIO recommends use the latest stable release of your preferred browser for the best experience in the MinIO Console.
```

Comment:
typo
```suggestion
MinIO recommends using the latest stable release of your preferred browser for the best experience in the MinIO Console.
```

Reply:
Perhaps also switch around the emphasis?
```suggestion
For the best experience in the MinIO Console, MinIO recommends using the latest stable release of your preferred browser.
```

Reply:
Riffing off your alternative suggestion.